hack/deploy-local.sh

```diff
@@ -53,7 +53,15 @@ fi
 
 FSS_V1A2=$(yq '.spec.template.spec.containers[]| select(.name == "manager") | .env[] | select(.name == "FSS_WCP_VMSERVICE_V1ALPHA2") | .value' "$YAML")
 if [ "${FSS_V1A2}" = "false" ]; then \
-  yq -i 'del(.spec.versions[] | select(.name == "v1alpha2"))' "$YAML"
+  # remove conversion webhooks from CRDs
+  yq -i 'del(select(.kind == "CustomResourceDefinition") | .spec.conversion | select(.strategy == "Webhook"))' "$YAML"
+  # set storage version to v1alpha1
+  yq -i '(select(.kind == "CustomResourceDefinition") | .spec.versions[] | select(.name == "v1alpha1").storage = true)' "$YAML"
+  # remove v1alpha2 versions from CRDs
+  yq -i 'del(select(.kind == "CustomResourceDefinition") | .spec.versions[] | select(.name == "v1alpha2"))' "$YAML"
+  # remove the v1alpha2 only CRD - virtualmachinewebconsolerequests
+  yq -i 'del(select(.kind == "CustomResourceDefinition") | select(.metadata.name == "virtualmachinewebconsolerequests.vmoperator.vmware.com"))'
```

Comment:
It would be nice to delete based on `spec.versions[]` being empty (as done in the cayman_photon script). This would avoid the need for additional updates when adding a new CRD in v1a2 later.

Reply:
Agreed!. Found a good way to remove it..